src/components/AElfWallet/AElfWallet.js

```diff
@@ -0,0 +1,232 @@
+import getDividends from '../../utils/getDividends';
+import getConsensus from '../../utils/getConsensus';
+import getWallet from '../../utils/getWallet';
+import getStateJudgment from '../../utils/getStateJudgment';
+import './AElfWallet.less';
+
+const RadioGroup = Radio.Group;
+
+export default class AElfWallet extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.walletInfo = JSON.parse(localStorage.walletInfoList);
+        if (this.walletInfo != null) {
+            this.wallet = getWallet(JSON.parse(localStorage.currentWallet).privateKey);
+        }
+        this.consensus = getConsensus(this.wallet);
+        this.dividends = getDividends(this.wallet);
+        this.state = {
+            walletInfo: this.walletInfo,
+            value: JSON.parse(localStorage.currentWallet).privateKey,
+            refresh: 0,
+            loading: false,
+            loadingTip: null
+        };
+    }
+
+    async componentDidMount() {
+        await this.pushWalletInfo();
+    }
+
+    pushWalletInfo = async () => {
+        let walletInfo = this.state.walletInfo;
+        if (walletInfo != null) {
+            for (let i = 0; i < walletInfo.length; i++) {
+                walletInfo[i].balance = parseInt(tokenContract.BalanceOf(walletInfo[i].address).return, 16) || '-';
+                walletInfo[i].dividends = parseInt(
+                    this.dividends.GetAllAvailableDividends(walletInfo[i].publicKey).return, 16
+                ) || '-';
+                walletInfo[i].tirkets = 0;
+                let tirkets = JSON.parse(
+                    hexCharCodeToStr(this.consensus.GetTicketsInfoToFriendlyString(walletInfo[i].publicKey).return)
+                ).VotingRecords;
+                if (tirkets !== undefined) {
+                    for (let j = 0; j < tirkets.length; j++) {
+                        if (tirkets[j].From === walletInfo[i].publicKey) {
+                            walletInfo[i].tirkets += parseInt(tirkets[j].Count, 10);
+                        }
+                    }
+                }
+            }
+        }
+        this.setState({
+            walletInfo,
+            refresh: this.state.refresh + 1
+        });
+    }
+
+
+    changeRadio(e) {
+        this.setState({
+            value: e.target.value
+        });
+        this.wallet = getWallet(e.target.value);
+        this.consensus = getConsensus(this.wallet);
+        this.dividends = getDividends(this.wallet);
+        for (let i = 0, len = this.state.walletInfo.length; i < len; i++) {
+            if (e.target.value === this.state.walletInfo[i].privateKey) {
+                localStorage.setItem('currentWallet', JSON.stringify(this.state.walletInfo[i]));
+                this.props.getCurrentWallet();
+            }
+        }
+    }
+
+
+    getWalletAssetInfo() {
+        const walletAssetInfo = this.state.walletInfo.map((item, index) => {
+            let balance = item.balance;
```

Comment:
let {balance, tikets, dividends} = item;

Reply:
Okay, thank you. I've been taught.